packages/gestalt/src/TextField.js

```diff
@@ -79,7 +79,7 @@ type Props = {|
   /**
    * The type of input. For non-telephone numerical input, please use [NumberField](https://gestalt.pinterest.systems/numberfield).
    */
-  type?: 'date' | 'email' | 'number' | 'password' | 'tel' | 'text' | 'url',
+  type?: 'date' | 'email' | 'password' | 'tel' | 'text' | 'url',
```

Comment:
Remove comment above. Doesn't apply anymore.

Reply:
The comment absolutely still applies — we want to guide users to NumberField.

Reply:
Then we are doing a Related section inside Prop tables. Here I'd update Textfield to link to the Variants section and add a comment there.

Reply:
Done